File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.py
class Solution:
    def reverseDegree(self, s: str) -> int:
        n = len(s)
        rDegree =0

        for i in range(1,n+1):
            product = (26 - (ord(s[i-1]) - ord('a'))) * i

            rDegree += product
        return rDegree

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.cpp
class Solution {
public:
    int reverseDegree(string s) {
        int n = s.length();

        int rDegree = 0;

        for(int i=1;i<=n;i++){
            int product = (26-(s[i-1]-'a')) * i ;

            rDegree += product;
        }
        return rDegree;
        
    }
};